articles/overview/index.md

```diff
@@ -19,7 +19,7 @@ You can connect any application (written in any language or on any stack) to Aut
 
 This video will give you a walkthrough of Auth0 and how it will help save you time adding identity to your application.
 
-<%= include('../videos/_video', { id: 's15ysw45uy' }) %>
+<%= include('../../_video', { id: 's15ysw45uy' }) %>
```

Comment:
The **Videos** page is still listed at the index (see last entry): https://auth0-docs-content-pr-6124.herokuapp.com/docs/getting-started